### ISSUE TITLE
Stop printing unused workspaces warning

### ISSUE THREAD

**Issue by Cynical-Optimist** [buildstream-migration/bst-staging#360]
[See original issue on GitLab](https://gitlab.com/BuildStream/buildstream/-/issues/360)
In GitLab by [[Gitlab user @TheRealMichaelCatanzaro]](https://gitlab.com/TheRealMichaelCatanzaro) on Apr 15, 2018, 20:04

I see this warning when building anything that's not epiphany:

```
[--:--:--][][] WARNING Unused workspaces

    core/epiphany.bst
    core/glib-networking.bst
```

Looks like it's telling me that I'm not building epiphany. I really don't need to be told that. Not sure what the value in having this warning is.

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Apr 15, 2018, 20:14

I've been thinking the same... I'm not sure there is any point to that, and it's especially misleading to have it as a "warning"

I'd still like run this by a few others before outright removing it.

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Apr 16, 2018, 13:52

mentioned in commit 827a12947bd8323fdfb7ec16233db963df965526

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Apr 16, 2018, 13:52

mentioned in merge request !400

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Apr 16, 2018, 14:24

closed via merge request !400

**Comment by Cynical-Optimist** [comment]
In GitLab by [[Gitlab user @tristanvb]](https://gitlab.com/tristanvb) on Apr 16, 2018, 14:24

closed via commit 827a12947bd8323fdfb7ec16233db963df965526
